castle/cms/theming.py

```diff
@@ -54,7 +55,33 @@
 styleslot_xpath = etree.XPath('//*[@id="style_slot"]')
 dynamic_grid_xpath = etree.XPath('//*[@dynamic-grid]')
 LAYOUT_NAME = re.compile(r'[a-zA-Z_\-]+/[a-zA-Z_\-]+')
-
+meaningless_dexterity_control_panel_urls = (
```

Comment:
why 'meaningless' ?

Reply:
maybe nonexistant would be a better name? these tiles don't exist (or something they rely on doesn't exist, i'm not 100% sure) for the context that /@@dexterity-types uses, so for that context only, afaict, there is log puke saying those tiles don't render or something, to the tune of several hundred lines every time you hit that endpoint

Reply:
please also put a brief comment about why they are 'meaningless' or 'nonexistent' as well -- might help whomever has to work with this code in the future

Reply:
k, will do